kernel/comps/virtio/src/device/input/device.rs

```diff
@@ -179,11 +179,16 @@ impl InputDevice {
         let size = field_ptr!(&self.config, VirtioInputConfig, size)
             .read_once()
             .unwrap();
-        let data: [u8; 128] = field_ptr!(&self.config, VirtioInputConfig, data)
-            // FIXME: It is impossible to call `read_once` on `[u8; 128]`. What's the proper way to
-            // read this field out?
-            .read()
-            .unwrap();
+        // FIXME: It is impossible to call `read_once` on `[u8; 128]`. What's the proper way to
+        // read this field out?
+        let mut data: [u8; 128] = [0; 128];
```

Comment:
Since only the first `size` bytes in the data field is valid, just copy only the first `size` bytes and directly to the destination buffer `out` (creating `data` is unnecessary).